### PR TITLE
zh-Hant: translation for src/generics.md

### PR DESCRIPTION
**Pull request by kuanhungchen**
Part of #684.

### CODE REVIEW

File: po/zh-Hant.po [hunk 1]
```diff
@@ -7257,14 +7257,15 @@ msgstr ""
...
-msgstr ""
+msgstr "# 泛型"
 
 #: src/generics.md:3
 msgid ""
 "Rust support generics, which lets you abstract an algorithm (such as "
 "sorting)\n"
 "over the types used in the algorithm."
 msgstr ""
+"Rust支援泛型，因此可以抽象化一個演算法（例如：排序）而不需要依賴特定型別。"
```
[the remark this answers is not in the document]
Comment (kuanhungchen) [reply po/zh-Hant.po]: Thanks! Just added the English name here.

Comment (edong) [inline po/zh-Hant.po]: zh-TW style:
* There should be a space between Chinese characters and English letters or numbers. -> `Rust 支援`
* Half-width parentheses `()` should be used instead of full-width parentheses `（）`, even when surrounding Chinese text or surrounded by Chinese text. There shouldn't be spaces between the parentheses and the text within them, but there should be spaces outside of them if adjacent to Chinese characters. -> `算法 (例如：排序) 而`

Comment (kuanhungchen) [reply po/zh-Hant.po]: Updated, PTAL. Thanks!

File: po/zh-Hant.po [hunk 2]
```diff
@@ -2,14 +2,15 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Comprehensive Rust 🦀\n"
 "POT-Creation-Date: \n"
-"PO-Revision-Date: 2023-05-26 15:20+0200\n"
-"Last-Translator: Martin Geisler <mgeisler@google.com>\n"
+"PO-Revision-Date: 2023-05-31 09:11+0800\n"
+"Last-Translator: Hank Chen <kuanhungchen@google.com>\n"
 "Language-Team: Chinese (Traditional) <zh-l10n@lists.linux.org.tw>\n"
+"Language: zh_TW\n"
```
Comment (hueich) [inline po/zh-Hant.po:8]: Is this change intended?  Is there a reference for why it's zh_TW instead of zh-Hant?

File: po/zh-Hant.po [hunk 3]
```diff
@@ -7257,14 +7258,16 @@ msgstr ""
 
 #: src/generics.md:1
 msgid "# Generics"
-msgstr ""
+msgstr "# 泛型 (Generics)"
```
Comment (hueich) [inline po/zh-Hant.po:7261]: Not sure if there's a style guide about whether to include the English name in titles.

File: po/zh-Hant.po [hunk 4]
```diff
@@ -7257,14 +7258,16 @@ msgstr ""
...
 
 #: src/generics.md:3
 msgid ""
 "Rust support generics, which lets you abstract an algorithm (such as "
 "sorting)\n"
 "over the types used in the algorithm."
 msgstr ""
+"Rust 支援泛型 (Generics)，因此可以抽象化一個演算法 (例如：排序) 而不需要依賴"
```
Comment (hueich) [inline po/zh-Hant.po:7269]: The "抽象化" should probably be after.  E.g.:

因此可以將一個演算法 (例如：排序) 抽象化而不需要依賴

File: po/zh-Hant.po (hunk 2 above)
Comment (kuanhungchen) [reply po/zh-Hant.po]: No. The "Language" and "X-Generator" fields were somehow generated by the Poedit.
They are removed now.

File: po/zh-Hant.po (hunk 3 above)
Comment (kuanhungchen) [reply po/zh-Hant.po]: Updated.

File: po/zh-Hant.po (hunk 4 above)
Comment (kuanhungchen) [reply po/zh-Hant.po]: Updated. Thanks!

File: po/zh-Hant.po (hunk 2 above)
Comment (kuanhungchen) [reply po/zh-Hant.po]: In some languages (e.g. ja, de) they include X-Generator while others (e.g. zh-Hans, el) don't. Not sure if we want to have it included.

Comment (mgeisler) [reply po/zh-Hant.po]: As far as I know, nothing in our pipeline uses the metadata here — we derive the URL directly from the name of the PO file. So you can change this as you like :slightly_smiling_face:

**Review by hueich — APPROVED** [review]
Looks good, thanks!

**Review by hueich — APPROVED** [review]
Looks good, thanks!

**Comment by kuanhungchen** [comment]
Thanks for all the review!
